math/genvector/doc/index.md

```diff
@@ -68,89 +71,83 @@ functionality.
 ## Concrete Vector typedefs
 
 To avoid exposing templated parameters to the users, typedefs are defined for all types of vectors based an `double`s and `float`s.
+The table below lists the `double` versions; the `float` counterpart ends on an extra `F`, such as ROOT::Math::XYPointF instead of ROOT::Math::XYPoint.
```

Comment:
Ah yes, that was something I wanted to ask but eventually forgot about...

Reply:
Yeah unless you came up with a good counter-argument I propose we simplify the list that way?

Reply:
No, was left for "future work" :smiley: